src/room_member.cpp

```diff
@@ -175,7 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    RakNet::SocketDescriptor socket(client_port,"");
+    RakNet::SocketDescriptor socket(client_port, "");
```

Comment:
I recognized that too. But in the documentation it states:
SocketDescriptor(unsigned short _port, const char *_hostAddress);
... Pass an empty string to use INADDR_ANY.